core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -667,7 +667,9 @@ public String toString() {
               "${%s}/core-site.xml:${%s}/hdfs-site.xml", Name.CONF_DIR, Name.CONF_DIR))
           .setDescription("Location of the HDFS configuration file to overwrite "
               + "the default HDFS client configuration. Note that, these files must be available"
-              + "on every node.")
+              + "on every node, but if there are a "
+              + Name.UNDERFS_HDFS_CONFIGURATION_PREFIX + " prefix, these files just need to be "
```

Comment:
This should explain that the hdfs configuration will be converted into mount properties, so that is why it only needs to be on the masters.

Reply:
Done.